erpnext/shopping_cart/product_info.py

```diff
@@ -21,9 +21,11 @@ def get_product_info_for_website(item_code, skip_quotation_creation=False):
 	if not skip_quotation_creation:
 		cart_quotation = _get_cart_quotation()
 
+	selling_price_list = cart_quotation.get("selling_price_list") if cart_quotation else _set_price_list(cart_settings, None)
+
```

Comment:
```suggestion
	if not skip_quotation_creation:
		cart_quotation = _get_cart_quotation()
		selling_price_list = cart_quotation.get("selling_price_list", None)
	
	if not selling_price_list:
		selling_price_list = _set_price_list(cart_settings, None)

```

More verbose, but easier to read. Also instead of `if not selling_price_list` we can add the code in an else block depending on the functionality